### DISCUSSION TITLE
Review of `rust-bitcoinconsensus` PRs

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#883] [General]
Could any folks willing, please add [rust-bitcoinconsensus](https://github.com/rust-bitcoin/rust-bitcoinconsensus) to your watch list. I have a few cricket patches sitting open there (PRs open for two months with no reviews or comments).

Thanks

**Comment by dr-orlovsky** [comment]
I am watching it, but not a maintainer there

**Comment by tcharding** [comment]
Annoyingly persistent bump: Please review/ack/nack/merge open PRs on `rust-bitcoinconsensus` crew. I keep raising this because the upcoming MSRV bump is stack wide. @apoelstra you are not listed as watching that repo, is that on purpose?

**Comment by apoelstra** [comment]
not intentional, i'll subscribe.

**Reply by tcharding** [reply]
Cheers mate.
